Britannia/localization/english/event_localization/BTWK_events_l_english.yml

```diff
@@ -1,25 +1,50 @@
+  BTWK_frontier_event.0202_desc_main:0 "\n\n [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] hopes that now the wall no longer serves any meaningful purpose, its stone could be repurposed for the developing settlements throughout [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] realm."
+  BTWK_frontier_event.0202_desc_imperial_like_destroyer:0 "[GetScope(destroyed_hadrians_wall_char).Char.GetFirstName] has done this despite [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] his heritage, from which is intrinsically linked to the momument. For this act, many of his kinsmen have been severely angered."
+  BTWK_frontier_event.0202_desc_else_destroyer:0 "Having no ancient ties to the land, and the wall which sits atop it, [GetScope(destroyed_hadrians_wall_char).Char.GetFirstNamePossessiveNoTooltipRegnal] decision was  tenable. The act has angered the likes of the Britons and of the Latins, who have ancient ties to the monument."
+  BTWK_frontier_event.0202.a_imperial_like:0 "A terrible day for Britannia"
+  BTWK_frontier_event.0202.a_other_0 "Interesting news" 
+  BTWK_frontier_event.0202.a_other_vassal:0 "For the good of the realm!"
+
+  BTWK_frontier_event.0203:0 "Hadrian's Wall Restored"  # ROOT of restoration
+  BTWK_frontier_event.0203_desc:0 "The conclusion of the renovations in [GetGlobalVar('last_old_hadrians_wall_province').MakeScope.Province.GetProvince] has marked the complete restoration of Hadrian's Wall. While the gesture was ultimately symbolic as a reminder of the Imperial roots all Britons share, the wall is now operable as the defensive frontline as it once was, should the need ever arise again. \n\nThough the effort to rebuild has taken many years, gold and manpower to complete, as the overseer of the final stones being set into place, you are the one the poems shall remember the most. Many among the isle are beginning to praise you as 'Hadrianus', as a successor to the great Emperor Hadrian himself. Perhaps this new reputation might aid your bids to become High King of Britannia, should you wish it..."
+  BTWK_frontier_event.0203.a:0 "OPTION"  # Not sure on this
+
+  BTWK_frontier_event.0204:0 "Hadrian's Wall Restored"  # Every other player 
+  BTWK_frontier_event.0204_desc:0 "[GetScope('restored_hadrians_wall_top_liege_char').Char.GetName)] has seen to the final parts Hadrian's Wall's restoration, returning dignity to the land in the wake of Rome's desertion. Many throughout Britannia have begun to praise him as '#italic Hadrianus#!', the heir to the great Roman Emporer Hadrian. Some even say that the completion of such a momentous feat gives [GetScope('restored_hadrians_wall_top_liege_char').Char.GetHerHim)] claim to Britannia's High Kingship... \n\n"
+  BTWK_frontier_event.0204_desc_vassal_contribution:0 "Despite being a lesser power within the realm of your liege,"  # If still vassal.
```

Comment:
Not entirely sure about this or the next line - the wording is kind of awkward and I don't totally think it's necessary

Reply:
Generally, they're for in the event that a player also upgrades segments of the wall while being under some top liege, so they don't miss out on any rewards. I'm essentially just trying to say "Your contributions are noted". The "former vassal" occurs in the event that there's an independence war/dissolution war (or if the current top liege dies and the realm is split) while the last slot is being upgraded. Though I suppose I could supplant all that with just a flat "Your own contributions to the wall's restoration have not gone unnoticed..."? I haven't started working on the triggers for the prior instances, so I don't mind scrapping 2 short lines of loc.

Reply:
Yeah, I was just thinking that "Your own contributions to the wall's restoration have not gone unnoticed" is probably sufficient for the player to understand what the event is getting at